src/UserFrontend.php

```diff
@@ -61,10 +64,26 @@ public static function login_head() {
    */
   public static function login_form_defaults(array $args) {
     $args['value_remember'] = TRUE;
+    $args['remember'] = FALSE;
     return $args;
   }
 
   /**
+   * @implements login_form
+   */
+  public static function login_form() {
+    ob_start(__CLASS__ . '::removesForgetmenotCheckbox');
```

Comment:
When you start an output buffer, you also need to end and clean it.

Do we actually need the output buffer?  Setting the `remember` parameter should hide the checkbox already?